src/hci/gnomex/security/ActiveDirectory.java

```diff
@@ -39,122 +38,123 @@
 import javax.naming.directory.SearchControls;
 import javax.naming.directory.SearchResult;
 
-import org.dom4j.Attribute;
+
 
 /**
  * Query Active Directory using Java
- * 
+ *
  * @filename ActiveDirectory.java
  * @author <a href="mailto:jeeva@myjeeva.com">Jeevanandam Madanagopal</a>
  * @copyright &copy; 2010-2012 www.myjeeva.com
  */
 public class ActiveDirectory {
-	// Logger
-	private static final Logger LOG = Logger.getLogger(ActiveDirectory.class.getName());
+    // Logger
+    //private static final Logger LOG = Logger.getLogger(ActiveDirectory.class.getName());

```

Comment:
The explicit class reference was a copy and paste.  It is in almost all of our servlets.  I will work on cleaning it up.  Same with the lower case log.  I will clean this up as well and make upper case.